demos/astro-playground/package.json

```diff
@@ -25,7 +25,7 @@
     "react": "^18.2.0",
     "react-dom": "^18.2.0",
     "svelte": "^5.0.0-next.1",
-    "typescript": "^5.2.2"
+    "typescript": "^5.4.0"
```

Comment:
Had to upgrade TS to get support for `Promise.withResolvers()` which is now maintstream.

Reply:
Reverted.